src/ResourceManager/Compute/Commands.Compute/help/Update-AzureRmVmss.md

```diff
@@ -191,7 +191,7 @@ The user identity references will be ARM resource ids in the form: '/subscriptio
 
 ```yaml
 Type: String[]
-Parameter Sets: (All)
+Parameter Sets: ExplicitIdentityParameterSet
```

Comment:
@hyonholee does this change break any customers who could be using the `IdentityId` parameter in the default parameter set?

Reply:
This was already fixed in the hotfix of last release.   IdentityId should be ExplicitIdentityParameterSet and SkuName shouldn't be.